website_v2/gcloud-powershell-website-v2/www/static/app/cmdletFilter.js

```diff
@@ -1,10 +1,10 @@
 ﻿var app = angular.module('powershellSite');
 
-/* Angular doesn't sort objects, only arrays, this filter was made.
+/** Angular doesn't sort objects, only arrays, so this filter was made.
```

Comment:
nit:
/**
  * Angular doesn't sort objects, only arrays, so this filter was made.